templates/includes/sidebar.html

```diff
@@ -0,0 +1,16 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">
+      Home page
+    </a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
```

Comment:
Replace the placeholder link `#` with an actual URL or a Django URL template tag for the Manufacturers page.